src/emoji-data-ts.ts

```diff
@@ -43,8 +43,9 @@ export const categoriesData = [
   'Flags'
 ]
 
-export const sheetColumns = 60
-export const sheetRows = 60
+export const sheetColumns = 61
+export const sheetRows = 61
+console.log("sheetColumns", sheetColumns);
```

Comment:
@miyanokomiya Could you remove `console.log`?

Reply:
@SatoshiKawabata I've fixed & rebased. Thanks!